Application.Models/DTOs/CreateCustomerRequest.cs

```diff
@@ -4,16 +4,16 @@
 using System.Text;
 using System.Threading.Tasks;
```

Comment:
Remover usings desnecessários, utilizar apenas System.
```suggestion
using System;
```